CHANGELOG.md

```diff
@@ -19,13 +20,43 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.1.0
+* ### Merged Pull Requests
+    * ...
+
+* ### Resolved Issues
+    * ...
+
+* ### Known Issues
+    * ...
+
 ## 1.0.0
 * ### Merged Pull Requests
-    *  ...
+    * Add support for strain calibration (#468 #469)
+    * Add support for DAQmx calibration info properties (#473)
+    * Add support for WaitForValidTimestamp to nidaqmx-python (#479)
+    * Fix raw-read Function Names Spacing Issue (#492)
+    * Update libtime (#504)
+    * Update AI Bridge Default Argument Values (#509)
+    * Improve error reporting in some cases (#550)
+    * Automate Driver Install Experience within nidaqmx Module (#512 #582)
+    * Rewrote examples (#566 #575 #573 #581 #583 #587 #570 #589 #592 #594)
+
+* ### Resolved Issues
+    * [38: No spacing in raw-read function names](https://github.com/ni/nidaqmx-python/issues/38)
+    * [384: Support internationalization](https://github.com/ni/nidaqmx-python/issues/384)
+    * [392: Indexing PhysicalChannelCollection fails for slices and strings containing a list/range of channels](https://github.com/ni/nidaqmx-python/issues/392)
+    * [482: Default argument values for bridge create channel functions are unusable](https://github.com/ni/nidaqmx-python/issues/482)
+
 * ### Major Changes
     * Add support for strain calibration (offset nulling and shunt calibration)
+    * Add support for DAQmx calibration info properties
+    * Add support for WaitForValidTimestamp
     * Fix naming issues:
         * Rename `ShuntCalSelect.AAND_B` to `A_AND_B`.
+    * Automate Driver Install Experience within nidaqmx Module
+    * Update libtime to accept time before 1970
+
```

Comment:
I don't think we should duplicate these items between "merged pull requests" and "major changes", and I think there are too many merged pull requests to list them all out individually.

For "merged pull requests", how about linking to https://github.com/ni/nidaqmx-python/compare/0.9.0...1.0.0 ?

Reply:
Agree, too many merged PRs to list out. It's updated as per suggestion.